pysat/instruments/pysat_testing_xarray.py

```diff
@@ -28,42 +35,70 @@
 
 
 # Init method
-init = mm_test.init
+def init(self, test_init_kwarg=None):
+    """Initialize the test instrument.
+
+    Parameters
+    ----------
+    self : pysat.Instrument
+        This object
+    test_init_kwarg : any or NoneType
```

Comment:
```suggestion
    test_init_kwarg : any
```

Looks like several of these were left over from @rstoneback's last big PR.  Probably worth grepping for this and not relying solely on my reading comprehension.

Reply:
added #1015 to find all of these throughout the package